miso-web/src/it/resources/it-context.xml

```diff
@@ -30,6 +30,11 @@
 
   <bean id="sessionFactory" class="org.springframework.orm.hibernate4.LocalSessionFactoryBean">
     <property name="dataSource" ref="dataSource" />
+    <property name="hibernateProperties">
+      <props>
+        <prop key="hibernate.dialect">org.hibernate.dialect.MySQL5InnoDBDialect</prop>

```

Comment:
Shouldn't need to specify this, but in the version of Hibernate we're using (4.2), we should probably use MySQL57InnoDBDialect if we do specify